File: backend/src/main/java/aron/utcn/licenta/dto/ReservationDto.java
package aron.utcn.licenta.dto;

import java.util.List;

import lombok.Data;

@Data
public class ReservationDto {

	private Integer id;
	private Integer userId;
	private Integer parkingSpotId;
	private String licensePlate;
	private String status;
	private String startTime;
	private List<Integer> duration;
}

File: mobile_app/app/src/main/java/com/example/licenta_mobile/dto/ReservationDto.kt
package com.example.licenta_mobile.dto

import java.time.LocalDate

class ReservationDto {
    var id: Int = 0
    var parkingSpotId: Int = 0
    lateinit var licensePlate: String
    lateinit var status: String
    lateinit var startTime: String
    lateinit var duration: List<Int>
}